cmd/ooniprobe/internal/database/actions.go

```diff
@@ -57,7 +57,7 @@ func GetMeasurementJSON(sess sqlbuilder.Database, measurementID int64) (map[stri
 		log.Errorf("failed to run query %s: %v", req.String(), err)
 		return nil, err
 	}
-	if measurement.IsUploaded {
+	if measurement.Measurement.IsUploaded {
```

Comment:
Now both `Measurement` and `Result` have `IsUploaded`. To disambiguate we need to explicitly specify that we want to select the `IsUploaded` attribute of the `Measurement`.